index.Rmd

```diff
@@ -1,4 +1,4 @@
---- 
+---
 title: "Getting used to R, RStudio, and R Markdown"
 author: "Chester Ismay"
```

Comment:
@pkq Please add yourself as co-author or editor here. I'm sure this took a ton of time and I'm very appreciative of the effort!

Reply:
Will do, thanks! Not as long as it took you, I'm sure!